### PR TITLE
Separation of Spring Boot Parent into multiple Versions

### PR DESCRIPTION
**Pull request by deveth0**
Please have a look into this MR. I tried to find a way which allows us to maintain multiple parents for different Spring Boot Versions. I think we should be fine with those subfolders, what do you think?

Do we need to maintain separate changes.xml for each parent or can we keep it in the root?

### CODE REVIEW

File: maven/spring-boot-parent/pom.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
   #%L
-  pro!vision GmbH
+  wcm.io
```
Comment (stefanseifert) [inline maven/spring-boot-parent/pom.xml]: please change back to pro!vision GmbH

File: maven/spring-boot-parent/pom.xml [hunk 2]
```diff
@@ -19,24 +19,28 @@
...
 
-  <name>de.pro-vision.maven.spring.spring-boot-parent</name>
-  <description>Parent for Spring Boot projects using the pro-vision parent as baseline.</description>
-  <url>https://github.com/pro-vision/pv-build-tools</url>
+  <name>Spring Boot Parent Root</name>
+
+  <modules>
+    <module>v2.1.x</module>
+    <module>v2.2.x</module>
+  </modules>
```
Comment (stefanseifert) [inline maven/spring-boot-parent/pom.xml:43]: the spring* and docker* should be removed in this file as well

**Review by stefanseifert — CHANGES_REQUESTED** [review]
bq. Do we need to maintain separate changes.xml for each parent or can we keep it in the root?
this depends if you want to release them independently, or always together. if idependently you should maintain a changes.xml for each one (and remove the shared one on this level)

**Comment by deveth0** [comment]
> bq. Do we need to maintain separate changes.xml for each parent or can we keep it in the root?
> this depends if you want to release them independently, or always together. if idependently you should maintain a changes.xml for each one (and remove the shared one on this level)

Ok, I added a second changes.xml.

**Comment by deveth0** [comment]
Tomorrow the new releases 2.1.11 and 2.2.2 will be released, I already updated the changes and would merge the PR tomorrow and then build a release.

**Review by stefanseifert — APPROVED** [review]

**Pull request merged**